language/move-core/types/src/vm_status.rs

```diff
@@ -679,6 +693,9 @@ pub enum StatusCode {
     STORAGE_WRITE_LIMIT_REACHED = 4027,
     MEMORY_LIMIT_EXCEEDED = 4028,
     VM_MAX_TYPE_NODES_REACHED = 4029,
+    EXECUTION_LIMIT_REACHED = 4030,
```

Comment:
FWIW new status codes need to be flag protected as once they are used, we cannot roll back (had an SEV for this). I guess you are aware.